.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
 on:
+  push:
+    tags:
+      - 'v*'
+      - '!v*-RC*'
```

Comment:
Is `!` valid in this case?

Reply:
Should be valid based on this documentation https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-branches.

Reply:
OK, thanks for the docs reference